src/openpa/src/primitives/opa_sun_atomic_ops.h

```diff
@@ -135,5 +135,6 @@ static _opa_inline int OPA_swap_int(OPA_int_t * ptr, int val)
 
 /* is this portable enough? */
 #define OPA_compiler_barrier()   __asm__ __volatile__  (""  ::: "memory")
+#define OPA_compiler_barrier()   __asm__ __volatile__  ("pause"  ::: "memory")
```

Comment:
isn't this a duplication `#define`?

Reply:
Yes this has to be `OPA_pause` I think. This is a part of #3113 @wesbland